Specs/Core/Matrix3Spec.js

```diff
@@ -148,6 +148,24 @@ defineSuite([
         expect(returnedResult).toEqualEpsilon(expected, CesiumMath.EPSILON15);
     });
 
+    it('fromHeadingPitchRoll computed correctly', function() {
+        var sPi = Math.sin(CesiumMath.PI);
+        var cPi = Math.cos(CesiumMath.PI);
+        var sPiOver2 = Math.sin(CesiumMath.PI_OVER_TWO);
+        var cPiOver2 = Math.cos(CesiumMath.PI_OVER_TWO);
+        var s3PiOver2 = Math.sin(CesiumMath.PI_OVER_TWO * 3);
+        var c3PiOver2 = Math.cos(CesiumMath.PI_OVER_TWO * 3);
+
+        var headingPitchRoll = new HeadingPitchRoll(-CesiumMath.PI_OVER_TWO, -CesiumMath.PI, CesiumMath.PI_OVER_TWO * 3);
+        var expected = new Matrix3(cPi * cPiOver2, -c3PiOver2 * sPiOver2 + s3PiOver2 * sPi * cPiOver2, s3PiOver2 * sPiOver2 + c3PiOver2 * sPi * cPiOver2,
```

Comment:
Gotcha. That makes sense. Thanks! Let me know if you don't have time to get to it for a while, I can always compute an ugly case manually. Manual matrix multiplication is painful, but will keep me strong. :)